src/slack_hooks/service/scout.clj

```diff
@@ -0,0 +1,60 @@
+(ns slack-hooks.service.scout
+  (:require [clojure.data.json :as json]
+            [slack-hooks.slack :as slack]
+            [clojure.string :as string]))
+
+(def scout-username
+   (or
+     (System/getenv "SCOUT_USERNAME")
+     "Scout"))
+
+(def scout-avatar
+    (System/getenv "SCOUT_AVATAR"))
+
+(def scout-slack-url
+   (System/getenv "SCOUT_SLACK_URL"))
+
+(defn alert-color [alert]
+  (case alert
+    "start"  "danger"
+    "oneoff" "warning"
+    "end"    "good"
+    "warning"
+    ))
+
+(defn scout-message [request]
+  (let [payload         (-> request :params :payload)
+        data            (json/read-str payload :key-fn keyword)
+
+        server_name     (:server_name data)
+        server_hostname (:server_hostname data)
+
+        alert_status    (:lifecycle data) ; can be [start|end|oneoff]  -- "oneoff" is for alerts generated internally by plugins
+        alert_id        (:id data)
+        alert_message   (:title data)
+        alert_trigger   (:trigger data)
+        alert_time      (:time data)
+        alert_url       (:url data)
+        alert_name      (:plugin_name data) ; Plugin that's alerting
+
+        ]
+        {
+          :title (format "[Alert %s] %s on %s" alert_status alert_name server_name)
+          :title_link alert_url
+          :text (format "%s - %s" alert_name alert_message)
```

Comment:
Doesn't seem like we need to repeat `alert_name` here if we already got it on the title.